Airports/EGLL/SMR/Regions.txt

```diff
@@ -3189,2207 +3189,3 @@ N051.28.08.321 W000.26.05.358
 N051.28.01.879 W000.26.02.141
 
 ;END NEW HEATHROW
```

Comment:
```suggestion
```

Reply:
All changes requested have been completed in the latest commit.